### PR TITLE
Tree pose

### PR DESCRIPTION
**Pull request by wpdunk**
first check for tree pose implemented

"is either ankle above respective knee"

will iterate on this with further checks

feedback on if this is the right approach would be great before pressing on further. 

Thanks!

### CODE REVIEW

File: src/lib/posenetWrapper.test.js [hunk 1]
```diff
@@ -1,136 +1,194 @@
...
-  input = sample['incorrect'][1]
-  subject = new Pose(input)
-  expect(subject.isMountainPose()).toEqual(false)
-  input = sample['incorrect'][2]
-  subject = new Pose(input)
-  expect(subject.isMountainPose()).toEqual(false)
-  input = sample['incorrect'][3]
-  subject = new Pose(input)
-  expect(subject.isMountainPose()).toEqual(false)
+it("#isTreePose can recognise position", () => {
```
Comment (punchcafe) [inline src/lib/posenetWrapper.test.js]: good start but I think we need a couple more examples of successes and failures

File: src/lib/posenetWrapper.js [hunk 2]
```diff
@@ -1,63 +1,101 @@
...
     this.source = posenetObject;
     this._mm = new MathModels();
     this._bodypartIndexLookup = {
-      "nose" : 0, "leftEye" : 1, "rightEye" : 2,
-      "leftEar" : 3, "rightEar" : 4, "leftShoulder" : 5,
-      "rightShoulder" : 6, "leftElbow" : 7, "rightElbow" : 8,
-      "leftWrist" : 9, "rightWrist" : 10, "leftHip" : 11,
-      "rightHip" : 12, "leftKnee" : 13, "rightKnee" : 14,
-      "leftAnkle" : 15, "rightAnkle" : 16
+      nose: 0,
```
Comment (punchcafe) [inline src/lib/posenetWrapper.js]: this is gross. grosser than it already was

File: src/lib/posenetWrapper.js [hunk 3]
```diff
@@ -1,63 +1,101 @@
...
   }
 
-  _isPointBetween(point,boundary){
-    boundary.sort(function(a,b){return a - b});
-    const withinUpperBound = boundary[1] > point
-    const withinLowerBound = boundary[0] < point
-    return withinUpperBound && withinLowerBound
+  isTreePose() {
+    const c1 =
+      this.bodypart("rightAnkle").position["y"] >
```
Comment (punchcafe) [inline src/lib/posenetWrapper.js]: So here you're just checking that neither foot is raised above the respective knee, right?

**Comment by punchcafe** [comment]
Really good start. Looks like you've picked up with Asana Wrap functionality and testing really easily and quickly. The linting makes this pull request particularly huge though, but that's my bad really. Since it's not ready yet, I'm going to request further changes

File: src/lib/posenetWrapper.js (hunk 2 above)
Comment (wpdunk) [reply src/lib/posenetWrapper.js]: how to resolve this?

**Pull request merged**